### ISSUE TITLE
prompts.forEach is not a function

### ISSUE THREAD

**Issue by helios1138** [snyk/snyk#172]
- `node -v`: v9.11.2
- `npm -v`: 5.6.0
- `snyk -v`: 1.88.1
- Command run: `snyk wizard`

```
/###/.config/yarn/global/node_modules/snyk/cli/commands/protect/wizard.js:220
  prompts.forEach(function (prompt) {
          ^

TypeError: prompts.forEach is not a function
    at inquire (/###/.config/yarn/global/node_modules/snyk/cli/commands/protect/wizard.js:220:11)
    at processPackageManager (/###/.config/yarn/global/node_modules/snyk/cli/commands/protect/wizard.js:76:12)
    at wizard (/###/.config/yarn/global/node_modules/snyk/cli/commands/protect/wizard.js:53:10)
    at /###/.config/yarn/global/node_modules/snyk/lib/hotload.js:16:21
    at Object.<anonymous> (/###/.config/yarn/global/node_modules/snyk/cli/index.js:27:23)
    at Module._compile (internal/modules/cjs/loader.js:654:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
    at Module.load (internal/modules/cjs/loader.js:566:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
    at Function.Module._load (internal/modules/cjs/loader.js:498:3)
```

**Comment by gablaxian** [comment]
Hi @helios1138. We're investigating this issue. To help us further could you please send us your `package.json` file to support@snyk.io (and make sure to remove any sensitive data!)

Many thanks.

**Comment by helios1138** [comment]
@gablaxian the issue is also reproducible on a new empty repo
```
mkdir test && cd test
yarn init
yarn add ms
snyk wizard
```
```json
{
  "name": "test",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "ms": "^2.1.1"
  }
}
```

**Comment by helios1138** [comment]
@gablaxian I think the bug is [here](https://github.com/snyk/snyk/blob/master/cli/commands/protect/wizard.js#L76):
should be
```js
return inquire([prompt], {})
```
instead of
```js
return inquire(prompt, {})
```

**Comment by gablaxian** [comment]
You are absolutely right. We've just determined this ourselves and we’ve already had a PR to fix it!

Thanks for bringing it to our attention. It should be merged in the near future :)

**Comment by joshje** [comment]
Now fixed in version [1.88.2](https://github.com/snyk/snyk/releases/tag/v1.88.2).
